x-pack/plugins/apm/public/components/app/Settings/AgentConfigurations/index.tsx

```diff
@@ -62,13 +99,38 @@ export function AgentConfigurations() {
             <EuiTitle size="s">
               <h2>
                 {i18n.translate(
-                  'xpack.apm.agentConfig.configurationsPanelTitle',
+                  'xpack.apm.agentConfig.configurationsPanel.title',
```

Comment:
What was there already was correct according to the [Kibana i18n guidelines](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#message-types).